src/widget/utils.js

```diff
@@ -89,6 +89,30 @@ export function getLabel( element ) {
 	return typeof labelCreator == 'function' ? labelCreator() : labelCreator;
 }
 
+/**
+ * Adds functionality to provided {module:engine/view/editableelement~EditableElement} to act as a widget's editable:
+ * * sets `contenteditable` attribute to `true`,
+ * * adds `ck-editable` CSS class,
+ * * adds `ck-editable_focused` CSS class when editable is focused and removes it when it's blurred.
+ *
+ * @param {module:engine/view/editableelement~EditableElement} editable
+ * @returns {module:engine/view/editableelement~EditableElement} Returns same element that was provided in `editable` param.
+ */
+export function toWidgetEditable( editable ) {
+	editable.setAttribute( 'contenteditable', 'true' );
```

Comment:
I think this and the following line should be done via `Template.apply()`. One of the reasons we decided to have a `Template` class was that DOM attrs should be touched via unified API.